sonar plugin fails for some JS parsing issue

[ERROR] Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:3.7.0.1746:sonar (default-cli) on project reactor: Error during parsing of generic test execution report '/home/runner/work/adobe-dx/adobe-dx/apps/admin/app/js_reports/test-reporter.xml'. Look at the SonarQube documentation to know the expected XML format. Line 3 of report refers to a file which is not configured as a test file: /home/runner/work/adobe-dx/adobe-dx/apps/admin/app/jcr_root/apps/dx/admin/clientlibs/configs/src/js/AdobeFontsConfig/__tests__/AdobeFontsConfig.test.js -> [Help 1]

see https://github.com/adobe/adobe-dx/runs/924783292?check_suite_focus=true

--- a/parent/main/pom.xml
+++ b/parent/main/pom.xml
@@ -368,6 +368,7 @@ ${bundle.additional.properties}
                 <sonar.coverage.jacoco.xmlReportPaths>target/site/jacoco-merged/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
                 <sonar.javascript.lcov.reportPaths>js_reports/coverage/lcov.info</sonar.javascript.lcov.reportPaths>
                 <sonar.testExecutionReportPaths>apps/admin/app/js_reports/test-reporter.xml</sonar.testExecutionReportPaths>
+                <sonar.test.inclusions>**/*.test.js</sonar.test.inclusions>
                 <sonar.verbose>true</sonar.verbose>
             </properties>
         </profile>

